### PR TITLE
Cycamore 1.5.5 LINUX ONLY

### PR DESCRIPTION
**Pull request by stompsjo**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

This is for the Cyclus micro-release 1.5.5. We are only implementing Linux since the OSX conda-forge for Cyclus is not currently working.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,14 +12,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 0
   skip: True  # [win or py<=27 or py>=37]
```
Comment (bam241) [inline recipe/meta.yaml]: - release python version constrain
- add skip for OSX

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,14 +12,16 @@ source:
...
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
     - cmake
     - pkg-config
+    - {{ compiler('cxx') }}
+  host:
```
Comment (bam241) [inline recipe/meta.yaml]: looks like that rerender messed that up

**Review by bam241 — CHANGES_REQUESTED** [review]
here is what I caught.

You probably have to rerender after you lift/change the skip restrictions to get the proper configs to run

**Review by bam241 — APPROVED** [review]

**Comment by stompsjo** [comment]
@conda-forge-admin, please rerender

**Comment by stompsjo** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (stompsjo) [reply recipe/meta.yaml]: This change has been made.

**Comment by bam241** [comment]
LGTM!

thx @stompsjo

**Pull request merged**